lua/wikis/commons/Infobox/Extension/TeamHistory/Manual.lua

```diff
@@ -38,14 +39,17 @@ function TeamHistoryManual.parse(args)
 		role = String.upperCaseFirst(role)
 	end
 
+	local leaveDate = TeamHistoryManual._parseDatesToYmd(displayDates.leave, args.estimated_end)
+
 	return {{
 		team = team,
 		role = role,
 		joinDate = TeamHistoryManual._parseDatesToYmd(displayDates.join, args.estimated_start),
 		joinDateDisplay = displayDates.join,
-		leaveDate = TeamHistoryManual._parseDatesToYmd(displayDates.leave, args.estimated_end),
+		leaveDate = leaveDate,
 		leaveDateDisplay = displayDates.leave,
 		reference = {},
+		noStorage = displayDates.leave and not leaveDate
```

Comment:
the store module already has a function that checks the leavedate, IMO we could move this there